server/repository/src/migrations/v1_02_01/invoice_update.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    #[cfg(feature = "postgres")]
+    const INVOICE_MODE: &'static str = "invoice_mode";
+    #[cfg(feature = "postgres")]
+    sql!(
+        connection,
+        r#"
+            CREATE TYPE {INVOICE_MODE} AS ENUM (
+            'STORE',
+            'DISPENSARY'
+            );
+        "#
+    )?;
+    #[cfg(not(feature = "postgres"))]
+    const INVOICE_MODE: &'static str = "TEXT";
+
+    sql!(
+        connection,
+        r#"
+            ALTER TABLE invoice ADD COLUMN mode {INVOICE_MODE};
```

Comment:
Because Postgres uses an `Enum` data type and sqlite only uses `Text` now... variable just changes which type the mode should be depending on if it's postgres or sqlite

```
#[cfg(feature = "postgres")]
    const INVOICE_MODE: &'static str = "invoice_mode";
    #[cfg(feature = "postgres")]
    sql!(
        connection,
        r#"
            CREATE TYPE {INVOICE_MODE} AS ENUM (
            'STORE',
            'DISPENSARY'
            );
        "#
    )?;
    #[cfg(not(feature = "postgres"))]
    const INVOICE_MODE: &'static str = "TEXT";
```